src/wasm-binary-reader-objdump.c

```diff
@@ -31,11 +32,21 @@ typedef struct Context {
   size_t current_opcode_offset;
   size_t last_opcode_end;
   int indent_level;
+  int print_details;
```

Comment:
I usually use `WasmBool` (and `WASM_{TRUE,FALSE}`) instead of int for boolean values